src/MLJMultivariateStatsInterface.jl

```diff
@@ -150,4 +150,1052 @@ metadata_pkg.(
+
+# Operations
+
+- `transform(mach, Xnew)`: Return predictions of the target given new
+  features `Xnew` having the same Scitype as `X` above.
+
+# Fitted parameters
+
+The fields of `fitted_params(mach)` are:
+
+- `projection`: Returns the projection matrix (of size `(d, p)`).
+  Each column of the projection matrix corresponds to a principal component.
+  The principal components are arranged in descending order of
+  the corresponding variances.
+
+# Report
+
+The fields of `report(mach)` are:
+
+- `indim`: Dimensions of the provided data.
+- `outdim`: Dimensions of the transformed result.
+- `principalvars`: The variance of the principal components.
+
+# Examples
+
+```
+using MLJ
+using LinearAlgebra
+
+KPCA = @load KernelPCA pkg=MultivariateStats
+
+X, y = @load_iris
+
+function rbf_kernel(length_scale)
+    return (x,y) -> norm(x-y)^2 / ((2 * length_scale)^2)
+end
+
+model = KPCA(maxoutdim=2, kernel = rbf_kernel(1))
+mach = machine(model, X) |> fit!
+
+projection = transform(mach, X)
+```
+
+See also
+[`PCA`](@ref), [`ICA`](@ref), [`FactorAnalysis`](@ref), [`PPCA`](@ref)
+"""
+KernelPCA
+"""
+$(MMI.doc_header(ICA))
+
+`ICA` is a computational technique for separating a multivariate signal into
+additive subcomponents, with the assumption that the subcomponents are
+non-Gaussian and independent from each other.
+
+# Training data
+
+In MLJ or MLJBase, bind an instance `model` to data with
+    mach = machine(model, X)
+
+Where
+
+- `X`: is any table of input features (eg, a `DataFrame`) whose columns
+  are of scitype `Continuous`; check the scitype with `schema(X)`
+
+# Hyper-parameters
+
+- `k::Int=0`: The number of independent components to recover, set automatically if `0`.
+- `alg::Symbol=:fastica`: The algorithm to use (only `:fastica` is supported at the moment).
+- `fun::Symbol=:tanh`: The approximate neg-entropy function, one of `:tanh`, `:gaus`.
+- `do_whiten::Bool=true`: Whether or not to perform pre-whitening.
+- `maxiter::Int=100`: The maximum number of iterations.
+- `tol::Real=1e-6`: The convergence tolerance for change in matrix W.
+- `mean::Union{Nothing, Real, Vector{Float64}}=nothing`: mean to use, if nothing (default)
+   centering is computed and applied, if zero, no centering, a vector of means can
+   be passed.
+- `winit::Union{Nothing,Matrix{<:Real}}=nothing`: Initial guess for matrix `W` either
+   an empty matrix (random initilization of `W`), a matrix of size `k × k` (if `do_whiten`
+   is true), a matrix of size `m × k` otherwise. If unspecified i.e `nothing` an empty
+   `Matrix{<:Real}` is used.
+
+# Operations
+
+- `transform(mach, Xnew)`: Return lower dimensional projection of the target given new
+  features `Xnew` having the same Scitype as `X` above.
+
+# Fitted parameters
+
+The fields of `fitted_params(mach)` are:
+
+ BUG: Does not have a projection class. It would also be cool to see the whitened
+matrix in fitted_params, to show how the covariance is the identity
+
```

Comment:
Bug noted, thank you! 

https://github.com/JuliaAI/MLJMultivariateStatsInterface.jl/issues/42